charts/opentelemetry/windows_exporter_installer/windows_exporter_installer.dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:latest
```

Comment:
It better not to use the latest release as it may contain bugs / security vulnerabilities